gazebo_ros/src/gazebo_ros_init.cpp

```diff
@@ -33,6 +33,13 @@
 #include <memory>
 #include <string>
 
+#ifndef GAZEBO_ROS_HAS_PERFORMANCE_METRICS
+#if (GAZEBO_MAJOR_VERSION == 11 && GAZEBO_MINOR_VERSION > 1) || \
+    (GAZEBO_MAJOR_VERSION == 9 && GAZEBO_MINOR_VERSION > 14)
```

Comment:
uncrustify doesn't like my indentation here

~~~
--- src/gazebo_ros_init.cpp
+++ src/gazebo_ros_init.cpp.uncrustify
@@ -38 +38 @@
-    (GAZEBO_MAJOR_VERSION == 9 && GAZEBO_MINOR_VERSION > 14)
+  (GAZEBO_MAJOR_VERSION == 9 && GAZEBO_MINOR_VERSION > 14)
~~~

I think it looks better like this, but it's not a big deal. Is it allowed to tell uncrustify to ignore this line, or should I just make the change?

Reply:
For indentation, I think you can use

```c
// *INDENT-OFF*
#if (GAZEBO_MAJOR_VERSION == 11 && GAZEBO_MINOR_VERSION > 1) || \
    (GAZEBO_MAJOR_VERSION == 9 && GAZEBO_MINOR_VERSION > 14)
// *INDENT-ON*
```

But subjectively, I would just do what uncrustify says to avoid the extra "INDENT" directives. I don't feel strongly one way or the other.

Reply:
I ended up going with 3a875ea